src/api/tests/feature_tests/steps/endpoint_lambda_mapping.py

```diff
@@ -42,6 +43,7 @@ def get_endpoint_lambda_mapping() -> ENDPOINT_LAMBDA_MAPPING:
             "Device?device_type={device_type}&foo={foo}": api.searchDevice.index,
             "Device?foo={foo}": api.searchDevice.index,
             "ProductTeam/{product_team_id}": api.readProductTeam.index,
+            "/ProductTeam/{product_team_id}/Product/{product_id}": api.readCpmProduct.index,
```

Comment:
This will need to go above the "ProductTeam/{product_team_id}" otherwise the tests will fail